lib/beaker/hypervisor/vsphere_helper.rb

```diff
@@ -106,8 +106,8 @@ def find_vms(names, connection = @connection)
   end
 
   def find_datastore(dc, datastorename)
-    datacenter = @connection.serviceInstance.find_datacenter(dc)
-    datacenter.find_datastore(datastorename)
+    datacenter = @connection.serviceInstance.find_datacenter(dc) || raise("datacenter #{dc} not found")
+    datacenter.find_datastore(datastorename) || raise("datastore #{datastorename} not found")
```

Comment:
Add `#{dc}` to this message as well to show the context ?

Reply:
good idea, added it